### ISSUE TITLE
soho pager-standalone: Disabled navigation button still fires click event

### ISSUE THREAD

**Issue by nilamthakur** [infor-design/enterprise#3445]
Navigation buttons of soho pager-standalone are disabled but click event is still being fired. 

**Describe the bug**
Navigation buttons of soho pager-standalone are disabled but click event is still being fired. This causes issues that page get navigate to the root of application.

**To Reproduce**
 1.Set up enterprise project ' https://github.com/infor-design/enterprise-ng.git’. 
 2.Check pager-standalone.demo example, In that if we have checked "Disable Previous Button" it will disable that button but click event is still begin fired and its navigate to root of application.

**Expected behavior**
 Disabled buttons should do nothing.

**Version**
ids-enterprise: 4.21.0-dev.20190729

**Screenshots**

**Platform**
 - Device (if applicable) [e.g. iPhone 6 or Samsung Galaxy S6]
 - OS Version: [e.g. Windows 10 or iOS 8]
 - Browser Name [e.g. chrome, safari, stock browser]
 - Browser Version [e.g. 22, 66.0.3359.181 (Official Build) (64-bit)]

**Additional context**

**Comment by bthharper** [comment]
Pretty sure this is a duplicate of https://github.com/infor-design/enterprise/issues/2170.  Which still tests as fixed.

**Comment by tmcconechy** [comment]
Duplicate of #2170

**Comment by tmcconechy** [comment]
I tried to verify this on http://localhost:4200/ids-enterprise-ng-demo/pager-standalone and it did not seem to fire the click event as expected with that fix (4.25.1 / 6.4).

But also you could use the events for each page change vs the click. (Possible workaround on older versions but unverified)
```
        (firstPage)="onFirstPage($event)"
        (previousPage)="onPreviousPage($event)"
        (nextPage)="onNextPage($event)"
        (lastPage)="onLastPage($event)"
```
